### DISCUSSION TITLE
🎨  [Simulateur auto-entrepreneur] Des personnes ne répondent pas aux questions de la simulation

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2612] [Idées]
### Description de la proposition

C'est un problème remonté par Agnès Nardon lors de nombreuses entrevues avec des auto-entrepreneurs sur les salons. Certain⋅e⋅s ne ignorent les questions qui s'affichent en dessous après la saisie, a fortiori sur mobile. Ces personnes s'arrêtent à la première estimation de revenu, qui peut être très éloignée de la réalité. 

C'est un problème UX majeur, il faudrait donc trouver une solution ergonomique pour y répondre, de préférence sans compromettre la simplicité du service pour les personnes qui n'ont pas ce problème. 

Par ailleurs, un retour d'une amie à moi sur l'utilisation du simulateur salarié montrait clairement une incompréhension au niveau de ce qu'apporte ou pas la réponse aux questions. Elle pensait ainsi qu'il fallait forcément répondre à toutes les questions pour utiliser l'outil. Elle passait donc uniquement par mon-entreprise quand elle voulait une simulation complète, lui préférant d'[autres sites](https://www.salaire-brut-en-net.fr/) pour les simples conversions brut / net, ce qui est dommage.

### URL de la page

https://mon-entreprise.urssaf.fr/simulateurs/auto-entrepreneur

**Comment by liliced** [comment]
Obsolète avec #3136 (on espère :crossed_fingers:)
